configs/atom.ini

```diff
@@ -0,0 +1,11 @@
+[blog]
+name = CC technical blog
+source_path = /
+url_path = /feed.xml
```

Comment:
Could you make the URL path `/blog/feed.xml` since it is the feed for `/blog/` and not the whole site?

Reply:
For testing, I deployed a copy of the site on GitHub pages. https://makkoncept.github.io/testing/feed.xml

Reply:
done